sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -115,17 +119,27 @@ describe("LocalWorkspace", () => {
         };
         let workspace: LocalWorkspace;
         beforeEach(async () => {
-            workspace = await LocalWorkspace.create({
-                projectSettings: projectSettings,
-            });
+            workspace = await LocalWorkspace.create(
+                withTestBackend({
+                    projectSettings: projectSettings,
+                }),
+            );
             await workspace.createStack(stackName);
         });
         it("lists tag values", async () => {
+            if (!process.env.PULUMI_ACCESS_TOKEN) {
+                // Skip the test because the local backend doesn't support tags
+                return;
```

Comment:
Out of curiosity, I had a quick look, and it seems we could also use `this.skip()` here to properly mark it as skipped, but apparently that doesn't work in async tests before mocha 3, and we use 2 🙃  https://mochajs.org/#inclusive-tests
(would also use `function() {...}` instead of `() => {}`.

We could potentially add a `console.log(`skipping ...`) in these cases, but it's probably not necessary.

Reply:
I didn't look into why it didn't work tbh, I just noticed it didn't, thanks for that explanation!  I added the `console.log`, seems useful!